queries.sql

```diff
@@ -8,3 +8,89 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* update the animals table by setting the species column to unspecified */
+
+/* Begin Transaction A */
+BEGIN;
+
+update animals set species = 'unspecified';
+
+/* Verify that change was made */
+SELECT * from animals;
+
+/* Rollback changes */
+ROLLBACK;
+
+/* Verify changes */
+SELECT * from animals;
+
+/* Begin Transaction B */
+BEGIN;
+
+/* update species col for animals with names ending in mon */
+update animals set species = 'digimon' where name like '%mon';
+
+/* update species col for animals with names not ending in mon */
+update animals set species = 'pokemon' where species IS null;
+
```

Comment:
- [ ] Good job on this queries but please kindly include a query to confirm the update was made before committing.

Reply:
Hello @AnselemOdims , its already confirmed as shown from the screenshots above
![image](https://user-images.githubusercontent.com/47685927/207586215-a5a6b631-4bae-4d63-9d0b-a3785ff2ae9c.png)